### ISSUE TITLE
Support Profiling

### ISSUE THREAD

**Issue by WestleyArgentum** [forio/julia-studio#134]
Expose profiling built into the language: http://docs.julialang.org/en/latest/stdlib/profile/

Possibly a good way to do this would be to integrate [ProfileView.jl](https://github.com/timholy/ProfileView.jl)

**Comment by ViralBShah** [comment]
Would it be possible for Julia Studio to highlight the results of the profiler in the editor, the way matlab does it?

**Comment by WestleyArgentum** [comment]
Yes, and I think that should absolutely be the direction we head in
